### PR TITLE
Add support to filter releases by state in the JSON API

### PR DESCRIPTION
**Pull request by cverna**
This will be useful for fedora-packages to get the list of 'active' releases . Currently it is getting all the releases and checking the state attribute of each release.

### CODE REVIEW

**Review by pypingou — DISMISSED** [review]

File: bodhi/tests/server/services/test_releases.py [hunk 1]
```diff
@@ -195,6 +195,35 @@ def test_new_release(self):
 
         self.assertEquals(r.state, ReleaseState.disabled)
 
+    def test_list_releases_by_current_state(self):
+        """ Test that we can filter releases using the 'current' state """
+        res = self.app.get('/releases/', {"state": 'current'})
+        body = res.json_body
+        print(len(body['releases']))
```
Comment (bowlofeggs) [inline bodhi/tests/server/services/test_releases.py]: Let's drop this print statement.

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
Looks good, just a stray print statement to drop and we should be good.

**Comment by cverna** [comment]
Updated :sake:

**Review by bowlofeggs — APPROVED** [review]

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
These patches are planned to be included in the upcoming 3.10.0 release: #2556.

**Comment by bowlofeggs** [comment]
These patches have been deployed to Fedora's staging Bodhi instance:

https://bodhi.stg.fedoraproject.org/

**Pull request merged**